content/en/docs/collector/configuration.md

```diff
@@ -3,12 +3,8 @@ title: "Configuration"
 weight: 20
 ---
 
-Please be sure to review the following documentation:
-
-- [Data Collection concepts](../../concepts/data-collection) in order to
-  understand the repositories applicable to the OpenTelemetry Collector.
-- [Security
-  guidance](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security.md)
```

Comment:
I think this file was just renamed to https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md ?

Reply:
Oh, heh, for some reason I didn't see this as being in the collector repo rather than docs.